pkg/telemetry/types/types.go

```diff
@@ -5,7 +5,7 @@ package types
  */
 
 type SyncerTelemetryConfig struct {
-	Disabled           string `json:"disabled,omitempty"`
+	Disabled           *bool  `json:"disabled,omitempty"`
```

Comment:
Why is this a pointer?

Reply:
I think its was initially set as string->bool ([here](https://github.com/loft-sh/vcluster/blob/main/pkg/telemetry/collect.go#L44)) and since since I've converted it to a bool now, keeping it a pointer would help us distinguish it from not set vs. explicitly set to `false` [here](https://github.com/ishankhare07/vcluster/blob/values-struct-and-validation/pkg/telemetry/collect.go#L44-L45)

Reply:
But we just need to check if its `true` right? Or do we have any check where we explicitly want to distinguish between `false` and `unset`?